app/routes/questionnaire.py

```diff
@@ -302,15 +301,11 @@ def _generate_wtf_form(block_schema, schema, current_location):
 
 
 def submit_answers(schema, questionnaire_store, full_routing_path):
-    message = json.dumps(
-        convert_answers(schema, questionnaire_store, full_routing_path), for_json=True
+    submission_handler = SubmissionHandler(
+        schema, questionnaire_store, full_routing_path
     )
-    submission_handler = SubmissionHandler(questionnaire_store.metadata)
-    submission_handler.submit_message(message)
-
-    get_questionnaire_store(current_user.user_id, current_user.user_ik).delete()
-
-    return redirect(submission_handler.get_next_location_url())
+    submission_handler.submit_questionnaire()
+    return redirect(url_for("post_submission.get_thank_you"))
```

Comment:
I still think it would be clearer to instantiate these in the route, as it not much duplication. But I if we are happy keeping it like this, I think the wrapper method should be renamed to `submit_questionnaire` or `handle_submission` since it is both submitting and redirecting.

Reply:
I actually agree, and did do it separately initially as I felt the same. I will revert as you have asked twice and there is a chance you might remind me in the future when I have long forgotten

Reply:
Waiting for @ajmaddaford to comment to revert :trollface: